che-plugin-openshift-ext-client/src/main/resources/org/eclipse/che/ide/ext/openshift/client/OpenshiftLocalizationConstant.properties

```diff
@@ -121,8 +121,14 @@ deploy.project.window.variables.add.tooltip=Add new variable
 deploy.project.window.labels.add.tooltip=Add new label
 deploy.project.success=Project <b>{0}</b> successfully deployed to OpenShift
 
-invalid.openshift.project.name.error=Name must be a valid DNS label
-invalid.che.project.name.error=Name must be a valid Che project name
+invalid.application.name.error=Invalid application name (?)
+invalid.application.name.detail.error=Application names must be in range between 2 and 23 symbols, and may only contain lower-case letters, numbers, and dashes. They may not start or end with a dash.
+invalid.project.name.error=Invalid project name (?)
+invalid.project.name.detail.error=Project names must be in range between 2 and 63 symbols, and may only contain lower-case letters, numbers, and dashes. They may not start or end with a dash.
+invalid.variables.error=There are errors in variable list
+invalid.labels.error=There are errors in label list (?)
+invalid.labels.detail.error=A valid object label has the form [domain/]name where a name is an alphanumeric (a-z, and 0-9) string, with a maximum length of 63 characters, with the '-' character allowed anywhere except the first or last character. A domain is a sequence of names separated by the '.' character with a maximum length of 253 characters.
```

Comment:
Write this in multiline mode. Example of it you can take a look by https://github.com/codenvy/che-plugins/blob/d67a22679d47a017ffba2fc07bdc1ad53f6cba24/plugin-sdk/che-plugin-sdk-env-local/src/main/resources/org/eclipse/che/env/local/client/LocalizationConstant.properties#L12-L13